{
    "status_code": 15,
    "lang": "python3",
    "run_success": false,
    "runtime_error": "Line 2: IndentationError: expected an indented block after class definition on line 51",
    "full_runtime_error": "IndentationError: expected an indented block after class definition on line 51\n    ^\n    def threeSum(self, nums: List[int]) -> List[List[int]]:\nLine 2  (Solution.py)",
    "status_runtime": "N/A",
    "memory": 8972000,
    "question_id": "15",
    "elapsed_time": 25,
    "compare_result": "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
    "code_output": "",
    "std_output": "",
    "last_testcase": "[-1,0,1,2,-1,-4]",
    "expected_output": "[[-1,-1,2],[-1,0,1]]",
    "task_finish_time": 1648625631892,
    "total_correct": 0,
    "total_testcases": 318,
    "runtime_percentile": null,
    "status_memory": "N/A",
    "memory_percentile": null,
    "pretty_lang": "Python3",
    "submission_id": "670219332",
    "status_msg": "Runtime Error",
    "state": "SUCCESS"
}
